src/plugins/filter/_screen-lg-min.scss

```diff
@@ -1,6 +1,6 @@
 //Screen size below medium
 .wb-filter {
 	.input-group {
-		max-width: 60%;
+		max-width: 702px;
```

Comment:
@duboisp The percentage does not work because when you wrap the filter in a smaller column, the search bar becomes very small when it should take the full-width of the column. The has been annoying for a while, but never got around to fix it. Thank you @fsnoddy for this.